website/blog/2023-05-30-0.72-metro-package-exports-symlinks.md

```diff
@@ -0,0 +1,173 @@
+- [Mitigation for TextInput ANRs on Android](/blog/2023/05/30/0.72-metro-package-exports-symlinks#react-native-package-renames)
+
+### Breaking Changes
+
+- [Deprecated Component Removals](/blog/2023/05/30/0.72-metro-package-exports-symlinks#deprecated-component-removals)
+- [Package renames](/blog/2023/05/30/0.72-metro-package-exports-symlinks#package-renames)
+
+<!--truncate-->
+
+## Highlights
+
+### Metro
+
+#### Symlink support (Beta)
+
+Symlink support continues to be one of the top-requested features in Metro and for React Native 0.72, Metro introduces pre-stable support for resolving symlinks.
+
+Symlink support enables React Native to work transparently with monorepo setups and pnpm, removing the need for workarounds. See [Enabling pre-stable features](/blog/2023/05/30/0.72-metro-package-exports-symlinks#enabling-pre-stable-features) to use in your app.
+
+#### Package Exports support (Beta)
+
+[Package Exports](https://nodejs.org/api/packages.html#exports) is the modern alternative to the package.json "main" field and provides new capabilities for npm packages to define their public API and target React Native.
+
+By enabling Package Exports support in your Metro config, your app will be compatible with the wider JavaScript ecosystem, including via the new "react-native" community condition. See [Enabling pre-stable features](/blog/2023/05/30/0.72-metro-package-exports-symlinks#enabling-pre-stable-features) to use in your app.
+
+:::tip
+See the newly released [blog post](https://reactnative.dev/blog/2023/05/08/package-exports-support) to learn more about this feature and our plans for stable rollout.
+:::
+
+#### New Metro config file setup
+
+In React Native 0.72, we’ve changed the config loading setup for Metro in React Native CLI. Please update your project’s `metro.config.js` file to match the [template’s version](https://github.com/facebook/react-native/blob/76a42c292de838a0dd537935db792eaa81410b9b/packages/react-native/template/metro.config.js).
+
+:::info
+Please update your config file to following [format](https://github.com/facebook/react-native/blob/76a42c292de838a0dd537935db792eaa81410b9b/packages/react-native/template/metro.config.js). Changes to `metro.config.js` will become required in 0.73. For 0.72, we will log a warning if not updated.
+:::
+
+This moves control over extending the base React Native Metro config into your project, and we’ve cleaned up the leftover defaults. In addition, this means that standalone Metro CLI commands, such as `metro get-dependencies`, will now work.
+
+#### Enabling pre-stable features
+
+To enable these new Metro features in your project, update your app’s `metro.config.js` file and set either the `resolver.unstable_enableSymlinks` or `resolver.unstable_enablePackageExports` options.
+
+```js
+const config = {
+  // ...
+  resolver: {
+    unstable_enableSymlinks: true,
+    unstable_enablePackageExports: true,
+  },
+};
+```
+
+### Developer Experience Improvements
```

Comment:
the "Better error readability", "Faster compilation and JSON parsing in Hermes" and "More ECMAScript support in Hermes" sections are all Hermes specific, can we move into their own section?

Reply:
Hermes is the default engine at this point. I'd want to communicate these changes as general devX improvements.

Reply:
sure - but that could be said for Metro too; why does Metro have its own section and Hermes doesn't?

Reply:
Hmm, I see your point. I'd say it's because this release is highlighting the features added in Metro and also one that requires manual updates. The changes in Hermes are not stand-out highlights of this release. If the case was reversed, then we'd also create a highlight section about the Hermes features.

Put another way, the Metro section is not really about Metro, it's about call-out features/changes that are added in Metro. 

Thinking about it more, we could move the metro.config changes in the breaking changes section

Reply:
And maybe rename the title to "New Metro Capabilities" or similar